.gitignore

```diff
@@ -37,6 +37,9 @@ yarn.lock
 ui/coverage
 ui/stats.html
 ui/.frontend-gradle-plugin
+ui/public/vscode/extensions/*
+!ui/public/vscode/extensions/*.vsix
+!ui/public/vscode/extensions/yaml
```

Comment:
unfortunately yaml extension vsix includes node_modules which is too heavy so I kept the old way of having raw extension files :/